app/views/hierarchical_responses/answers/_nodes.html.erb

```diff
@@ -19,4 +26,20 @@
       <%= render("hierarchical_responses/answers/group_set", group_set: node, context: new_context) %>
     <% end %>
   </div>
+
+  <%# Setup condition js. %>
+  <%# Note that we need to do this in any form mode, even show. %>
```

Comment:
This is no longer true. Can you set a condition here only set this up on `new`? `edit` will come later, and `show` will never come :D

Reply:
`setup_condition_computer` is only called in the `new` controller action so the condition below should limit it to `new`-only as is